pkg/apis/kubeadm/v1beta1/types.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package v1beta1
 
 import (
-	v1 "k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
This package is copied from the upstream, not written by us so I'd like to avoid changing them.

Reply:
No problemo, fixed.